app/components/npq_separation/admin/statement_selector_component.html.erb

```diff
@@ -47,5 +47,17 @@
     </div>
   </div>
 
+  <div class="govuk-grid-row">
```

Comment:
I don't think we should be creating empty row classes? I think the unless should be up a level